modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -31,13 +31,16 @@
 
 	--n-tabs-title-color: var( --e-global-color-secondary, var( --n-tabs-color-secondary-fallback ) );
```

Comment:
Comment: I had to branch of this task from https://elementor.atlassian.net/browse/ED-8915, to be able to check the full title alignment behaviour.